core/layout_api.php

```diff
@@ -899,6 +900,39 @@ function layout_options_for_sidebar( $p_menu_options, $p_active_sidebar_page ) {
 	}
 }
 
+/**
+ * Print dropdown menu item
+ * @param array  $p_menu_options menu item of group
+ * @return void
+ */
+function layout_sidebar_dropdown_menu( $p_menu_options ) {
+	echo '<li class="hover">' . "\n";
+   	echo '<a href="#">' . "\n";
+   	echo '<i class="menu-icon fa fa-plug"></i>' . "\n";
+   	echo '<span class="menu-text">' . lang_get( "plugins" ) . '</span>' . "\n";
+   	echo '<span class="caret"></span>';
+   	echo '</a>' . "\n";
+	echo '<b class="arrow"></b>';
+        echo '<ul class="submenu" >';
+	foreach( $p_menu_options as $t_menu_option ) {
+		
+			$t_icon = isset( $t_menu_option['icon'] ) ? $t_menu_option['icon'] : 'fa-plug';
+			if( !isset( $t_menu_option['url'] ) || !isset( $t_menu_option['title'] ) ) {
+				continue;
+			}
+
+			if( isset( $t_menu_option['access_level'] ) ) {
+				if( !access_has_project_level( $t_menu_option['access_level'] ) ) {
+					continue;
+				}
+			}
+
+			layout_sidebar_menu( $t_menu_option['url'], $t_menu_option['title'], $t_icon, $p_active_sidebar_page );
```

Comment:
SYSTEM NOTICE: 'Undefined variable: p_active_sidebar_page' in '/media/sdb/www/html/git/mantis2/core/layout_api.php' line 930

Reply:
I corrected this.